components/chat/header/model-selector.tsx

```diff
@@ -3,28 +3,28 @@
  */
 
 import { ChatContext } from '@/app/chat-context';
-import { Select, Option, SelectOption } from '@/lib/mui';
+import { ModelType } from '@/types/chart';
+import { Select } from 'antd';
 import { ICON_MAP } from '@/utils/constants';
 import classNames from 'classnames';
 import Image from 'next/image';
 import { useContext } from 'react';
 import { useTranslation } from 'react-i18next';
 
 interface Props {
-  size: 'sm' | 'md' | 'lg';
```

Comment:
if you remove the `size` prop, then you should make sure that all references are properly styled.